package.json

```diff
@@ -22,35 +22,35 @@
     "@typescript-eslint/typescript-estree": "1.10.2",
     "angular-estree-parser": "1.1.5",
     "angular-html-parser": "1.2.0",
-    "camelcase": "4.1.0",
-    "chalk": "2.1.0",
+    "camelcase": "5.3.1",
+    "chalk": "2.4.2",
     "cjk-regex": "2.0.0",
-    "cosmiconfig": "5.0.7",
-    "dashify": "0.2.2",
+    "cosmiconfig": "5.2.1",
+    "dashify": "2.0.0",
     "dedent": "0.7.0",
-    "diff": "3.5.0",
-    "editorconfig": "0.15.2",
+    "diff": "4.0.1",
+    "editorconfig": "0.15.3",
     "editorconfig-to-prettier": "0.1.1",
-    "escape-string-regexp": "1.0.5",
+    "escape-string-regexp": "2.0.0",
     "esutils": "2.0.2",
     "find-parent-dir": "0.3.0",
     "find-project-root": "1.1.1",
     "flow-parser": "0.84.0",
-    "get-stream": "3.0.0",
+    "get-stream": "5.1.0",
```

Comment:
I think get-stream version `5.1.0` requires Node.js >= 8: https://github.com/sindresorhus/get-stream/blob/v5.1.0/package.json#L12-L14

Reply:
This is not a  problem, cause this package willbe transformed in  finale bundle